pygyw/__init__.py

```diff
@@ -6,10 +6,6 @@
 
 """
 
-__version__ = "1.3.0"
-__author__ = 'Antoine Malherbe'
-__credits__ = 'Get Your Way'
-
```

Comment:
This was done before because these infos were not accessible from the python code. Is it now possible ? It was useful especially for the version.

Reply:
Oh ok, I didn't see these variables used anywhere so that's why I removed them. But ok, I can add `__version__` back in case you want to use it.

Reply:
> Oh ok, I didn't see these variables used anywhere so that's why I removed them. But ok, I can add __version__ back in case you want to use it.
I don't use it personally, but the package is public and aims to be used by other people who may have different needs.

Reply:
It seems strange that an underscored variable is part of the public api, but ok.